tests/unit/preprocessor/_regrid/test_regrid.py

```diff
@@ -265,50 +267,126 @@ def make_test_cube(shape):
     return cube
 
 
-def test_rechunk_on_increased_grid():
-    """Test that an increase in grid size rechunks."""
-    with dask.config.set({'array.chunk-size': '128 M'}):
+def make_test_cube_unstructured(shape):
+    data = da.empty(shape, dtype=np.float32)
+    cube = iris.cube.Cube(data)
+    if len(shape) > 2:
+        cube.add_dim_coord(
+            iris.coords.DimCoord(
+                np.arange(shape[0]),
+                standard_name='time',
+            ),
+            0,
+        )
+    lat_points = np.linspace(-90., 90., shape[-2], endpoint=True)
+    lon_points = np.linspace(0., 360., shape[-1])
+
+    cube.add_aux_coord(
+        iris.coords.AuxCoord(
+            np.broadcast_to(lat_points.reshape(-1, 1), shape[-2:]),
+            standard_name='latitude',
+        ),
+        (-2, -1),
+    )
+    cube.add_aux_coord(
+        iris.coords.AuxCoord(
+            np.broadcast_to(lon_points.reshape(1, -1), shape[-2:]),
+            standard_name='longitude',
+        ),
+        (-2, -1),
+    )
+    return cube
+
+
+def make_test_cube_irregular(shape):
```

Comment:
I think this defines an unstructured cube (using [our definitions](https://esmvaltool--2457.org.readthedocs.build/projects/ESMValCore/en/2457/recipe/preprocessor.html#grid-types)).

Reply:
Fixed now